core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -76,6 +79,12 @@ class AdminClient(val time: Time,
     response.groups().asScala.map(group => GroupOverview(group.groupId(), group.protocolType())).toList
   }
 
+  def getApiVersions(node: Node): util.Collection[ApiVersion] = {
+    val response = send(node, ApiKeys.API_VERSIONS, new ApiVersionsRequest.Builder()).asInstanceOf[ApiVersionsResponse]
+    Errors.forCode(response.errorCode()).maybeThrow()
+    response.apiVersions()
```

Comment:
so, if the server is down, or if the request times out, we will still get an output with all APIS showing 'Unsupported'. IF we didn't get a response, it might be better to just print the error (timeout, connection refused, etc.), and return.